test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+
+public class NameTest {
+
+    @Test
+    public void getDistanceOfTwoNames_twoNonEmpty_shouldReturnDifference() throws IllegalValueException{
```

Comment:
According to the coding convention, I think the third part of the method name should be either `returnFalse` or `returnTrue`. Therefore you should separate below methods into different test cases.